src/js/utils/dom.js

```diff
@@ -70,7 +68,7 @@ export const insertFirst = function(child, parent){
  * @type {Object}
  * @private
  */
-export const cache = {};
+const _data = {};
```

Comment:
Cool. Is that the common practice? Add an underscore only to private vars that are going to be accessible externally?